### PR TITLE
add keyboard shortcut for rawlist selection

### LINKED ISSUE #699: keyboard shortcut for list selection ?
Hi,
It would be great to add the possibility to select a choice in a list through a keyboard shortcut, for example :
```
$ node list.js
? What do you want to do? (Use arrow keys)
❯ [O]rder a pizza 
  Make a [r]eservation 
  ──────────────
  Ask for opening [h]ours 
  - Contact support (Unavailable at this time)
  Talk to the receptionist 
```
Typing "o" would select the first choice (order a pizza), typing "r" the second choice (make a reservation), and so on.

Could be useful in the case of a long list, no need to use the arrow keys...

### PR DESCRIPTION
**Pull request by phantom0424**

### CODE REVIEW

**Comment by phantom0424** [comment]
This is my first time contributing code to the open source library. ^_^

File: packages/inquirer/lib/prompts/rawlist.js [hunk 1]
```diff
@@ -146,6 +150,35 @@ class RawListPrompt extends Base {
...
     this.render();
   }
+
+  /**
+   * When user press up key
+   */
+
+  onUpKey() {
+    if (this.selected) {
+      this.selected--;
```
Comment (SBoudrias) [inline packages/inquirer/lib/prompts/rawlist.js]: We'll need to also account for the selected index going under or over the list length.

File: packages/inquirer/lib/prompts/rawlist.js [hunk 2]
```diff
@@ -146,6 +150,35 @@ class RawListPrompt extends Base {
...
+    } else {
+      this.selected = this.opt.choices.realLength - 1;
+    }
+    this.rl.line = (this.selected + 1).toString();
```
Comment (SBoudrias) [inline packages/inquirer/lib/prompts/rawlist.js]: Is reassigning the `rl.line` necessary? I don't think it'll be in this case.

**Comment by SBoudrias** [comment]
Hey @phantom0424, thanks for the PR. Always happy to welcome new contributors!

File: packages/inquirer/lib/prompts/rawlist.js (hunk 2 above)
Comment (phantom0424) [reply packages/inquirer/lib/prompts/rawlist.js]: I think the `rl.line` value should correspond to the selected index, otherwise it is not very strange?

**Comment by phantom0424** [comment]
That error job is because it has timed out.

**Comment by SBoudrias** [comment]
@phantom0424 I think you closed this PR by mistake right?

**Comment by phantom0424** [comment]
Yes, I found that I referenced to a mismatched issue #699  .

**Comment by SBoudrias** [comment]
Thanks!

**Pull request merged**